x-pack/solutions/observability/test/functional/apps/dataset_quality/dataset_quality_table.ts

```diff
@@ -50,6 +50,18 @@ export default function ({ getService, getPageObjects }: DatasetQualityFtrProvid
       // Install Integration and ingest logs for it
       await PageObjects.observabilityLogsExplorer.installPackage(pkg);
 
+      // Disable failure store for logs-*
+      await synthtrace.createComponentTemplate({
```

Comment:
nit: you could simplify and put that into the index template directly, but doesn't matter really

Reply:
I liked your suggestion. Addressed in [f3daa96](https://github.com/elastic/kibana/pull/230374/commits/f3daa9626dc56c91e916eb84bc6a0dafce53ac7d).